{"name": "test_check_redirect_n_count_times[2]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0 == 2\n  +0\n  -2", "trace": "test_redirect.py:29: in test_check_redirect_n_count_times\n    assert len(r.history) == n\nE   assert 0 == 2\nE     +0\nE     -2\n        n          = 2\n        r          = <Response [404]>\n        url        = 'https://httpbin.org/redirect/2'"}, "attachments": [{"name": "log", "source": "56a607a4-098d-4465-9281-42c6b1aaea3a-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "15328fbf-a23b-4c20-9c98-925f75d1b34b-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "n", "value": "2"}], "start": 1596531095175, "stop": 1596531097302, "uuid": "a09f7242-aa0f-45b1-beaf-e589ae714039", "historyId": "1215f68b843c11cfba1929badde86a8a", "testCaseId": "f07b42c17c576f1548b8a61317a6569e", "fullName": "test_redirect#test_check_redirect_n_count_times", "labels": [{"name": "suite", "value": "test_redirect"}, {"name": "host", "value": "S340"}, {"name": "thread", "value": "32140-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_redirect"}]}